client/src/components/common/Questionnaire/SingleChoiceQuestion.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Radio } from 'antd';
+
+const { Group: RadioGroup } = Radio;
+const SingleChoiceQuestion = props => {
+  const {
+    onChange,
+    options: { question, answers },
+    id,
+    value,
```

Comment:
If you wanted to be real cheeky, you could trick the Array into being an Object and use array position as a key to destructure with, then use destructuring assignment to give it a more meaningful name

General Example w/ [CodePen](https://codepen.io/leggomuhgreggo/pen/oNNmmqg?editors=0012): 
```js
const arr = [1, 2, 3]

const {
  0: first,
  1: second,
  2: third
} = arr

console.log(first, second, third) // 1 2 3
```


How it could be applied here:
```suggestion
    value: {0: firstValue }
```

Reply:
thanks greg good to it's really useful here